### PR TITLE
[W-9566536, W-9580101] Add a picklist option to application status field

### PR DESCRIPTION
**Pull request by vmarasow**
# Changes
## New Application Status Picklist Value
To help your recruiting and admissions team better track the status of an application, we added a new picklist value, Started, to the Application Status field (hed__Application_Status__c) on the Application object. If you're starting with this version of EDA, it's included automatically. If you started with an earlier version, add this new value to the Status field. For details, see [Add Custom Fields and Picklist Values](https://powerofus.force.com/s/article/EDA-Add-Custom-Picklist-Values).

# Issues Closed
[W-9566536](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000QXDQYA4/view) - R&A
[W-9580101](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000QlfCYAS/view) - EDA

# New Metadata
## Picklist Option
- Application -> Application Status: "Started"

# Testing Notes

### CODE REVIEW

File: force-app/main/default/objectTranslations/Application__c-ca/Application_Status__c.fieldTranslation-meta.xml [hunk 1]
```diff
@@ -47,4 +47,8 @@
         <masterLabel>Declined Offer</masterLabel>
         <translation>Oferta rebutjada</translation>
     </picklistValues>
+    <picklistValues>
```
Comment (RadSFDeveloper) [inline force-app/main/default/objectTranslations/Application__c-ca/Application_Status__c.fieldTranslation-meta.xml:50]: The new "Started" value is added at the end of picklist values. I am wondering if it should be added before "Submitted"? @MatthewBlanski , @vmarasow 
Screen Shot 2021-07-19 at 2 34 27 PM
Screen Shot 2021-07-19 at 2 36 16 PM

**Review by agoolsby — COMMENTED** [review]
@vmarasow  I updated the Release Notes (Changes area). FYI @juliesnow

**Review by agoolsby — APPROVED** [review]

File: force-app/main/default/objectTranslations/Application__c-ca/Application_Status__c.fieldTranslation-meta.xml (hunk 1 above)
Comment (RadSFDeveloper) [reply force-app/main/default/objectTranslations/Application__c-ca/Application_Status__c.fieldTranslation-meta.xml:50]: @JasonFitzy , can you please check into Application Status ordering with regards to where new value "Started" should be added? Thank you.

Comment (JasonFitzy) [reply force-app/main/default/objectTranslations/Application__c-ca/Application_Status__c.fieldTranslation-meta.xml:50]: @RadSFDeveloper Thanks for asking! EDA picklists are either in alphabetical order or in an order due to some "flow" the data speaks to. As you astutely pointed out when thinking about the flow of application status "Started" should be before "Submitted". I agree. Can we put it in that position? 

and sorry on the delay I normally do not get questions in GitHub. :)

Comment (RadSFDeveloper) [reply force-app/main/default/objectTranslations/Application__c-ca/Application_Status__c.fieldTranslation-meta.xml:50]: It is all good, there is no delay, thanks for the response @JasonFitzy !

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Please move the started picklist value before the others in the field file. Note that the position in the translations fille has no effect.

**Review by RadSFDeveloper — APPROVED** [review]
Looks good to me.

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
QA Passes - Merging

**Pull request merged**